extra/restart-datadog-agent.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [[ $DISABLE_DATADOG_AGENT ]]; then
+  echo "DISABLE_DATADOG_AGENT environment variable is set, not starting the agent."
+  exit 0
+fi
+
+# Update integration directory configuration
+sed -i -e"s/^.*UNIX_CONFIG_PATH =.*$/UNIX_CONFIG_PATH = '\/app\/.apt\/opt\/datadog-agent\/agent'/" /app/.apt/opt/datadog-agent/agent/config.py
```

Comment:
This brings the question: how do we manage the version of the datadog agent to install?  it looks like we are installing without mentioning a version. Will that break if our buildpack installs a different version and their code changes in a way that doesn't read this constant ?